src/Styles/Book.css

```diff
@@ -0,0 +1,150 @@
+.books {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 0.938rem;
+}
+
+.book {
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 2rem;
+}
```

Comment:
- [ ] Please 🙏🏽 use the correct height, width, margin, and padding defined in .Lesson-Panel from the original design to match it.